File: public/examples/integrations/toolkits/firecrawl/scrape_url_example_call_tool.py
from arcadepy import Arcade

client = Arcade()  # Automatically finds the `ARCADE_API_KEY` env variable

USER_ID = "{arcade_user_id}"
TOOL_NAME = "Firecrawl.ScrapeUrl"

tool_input = {"url": "https://example.com", "formats": "Formats.MARKDOWN"}

response = client.tools.execute(
    tool_name=TOOL_NAME,
    input=tool_input,
    user_id=USER_ID,
)
print(response)

File: public/examples/integrations/toolkits/firecrawl/scrape_url_example_call_tool.js
import { Arcade } from '@arcadeai/arcadejs';

const client = new Arcade(); // Automatically finds the `ARCADE_API_KEY` env variable

const USER_ID = '{arcade_user_id}';
const TOOL_NAME = 'Firecrawl.ScrapeUrl';

const toolInput = {
  url: 'https://example.com',
  formats: 'Formats.MARKDOWN',
};

const response = await client.tools.execute({
  tool_name: TOOL_NAME,
  input: toolInput,
  user_id: USER_ID,
});

console.log(response);
